content/1_General/Cpp_Command.mdx

```diff
@@ -502,6 +502,10 @@ default.
 	</Resource>
 </Resources>
 
+Note that USACO does not have a stack size limit, aside from the usual 256 MB
+memory limit. Therefore, code that crashes locally due to a stack overflow error
+may still pass on the USACO servers.
```

Comment:
rofl this does not sounds like good advice :p

Reply:
why?

Reply:
i mean if it crashes due to stackoverflow it might also TLE on the servers due to the large amount of recursion tbh

Reply:
I was referring to the case where it crashes locally due to not including the appropriate compiler options for setting the stack size.